tests/PeekTools.spec.ts

```diff
@@ -6,7 +6,8 @@ describe("getSortedValues()", () => {
   it("should return values as specified in the options", () => {
     const ret = getSortedValues(createBaseValues(defaultValues), { gdp: -1 })
     const lastRow = ret.pop()
-    lastRow.id.should.be.equal("gdp")
-    lastRow.effect.should.be.equal("-1.00")
+    ;(lastRow === undefined).should.not.be.false
```

Comment:
Könnte auch mit `should(lastRow).not.be.undefined()` gehen. Finde ich lesbarer.

Reply:
Ah! Ich hatte lastRow.should.not.be.undefined probiert. Ging nicht. Danke! Lerne immer mehr von Dir.